src/net.h

```diff
@@ -786,8 +798,12 @@ class CNode
 private:
     const NodeId id;
     const uint64_t nLocalHostNonce;
-    // Services offered to this peer
+
+    //! Services offered to this peer.
+    //! This is supplied by the parent CConman during peer connection
+    //! (CConman::ConectNode()) from its attribute of the same name.
```

Comment:
Could explain why this is `const`, i.e. impossible to change after the connection. And maybe also how this interacts with IBD or background indexes that are turned on when the chain is already synced.

Reply:
Good suggestion, done.